closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -68,13 +67,12 @@ private NativeCrypto() {
 
     @CallerSensitive
     public static NativeCrypto getNativeCrypto() {
-
         ClassLoader callerClassLoader = Reflection.getCallerClass().getClassLoader();
 
-        if ((callerClassLoader != null) && (callerClassLoader != VM.getVMLangAccess().getExtClassLoader())) {
-            throw new SecurityException("NativeCrypto");
-        }
-        return new NativeCrypto();
+        if ((callerClassLoader == null) || (callerClassLoader == VM.getVMLangAccess().getExtClassLoader()))
```

Comment:
Using `{ }` is preferred, to avoid future mistakes when adding more content to the body of the if statement.

Reply:
Fixed.